src/Pelagos/Util/UrlValidation.php

```diff
@@ -20,8 +20,12 @@ class UrlValidation
     public function validateUrl($url)
```

Comment:
Would we like to make follow location optional, by default true?

Reply:
Probably we can change it to optional, when there is a use case of not following redirects?

Reply:
Can't think of any really.